### PR TITLE
Adding the cvmfs version requirement

### PR DESCRIPTION
**Pull request by efajardo**
Per the ticket [SOFTWARE-3384](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3384) I am adding this.

### CODE REVIEW

**Comment by efajardo** [comment]
@DrDaveD and @brianhlin can you review this please?

**Review by brianhlin — COMMENTED** [review]
Hey @efajardo , I don't think we need this for the linked ticket. It's something @DrDaveD proposed but I don't think it's strictly necessary for shipping the osg-oasis and cvmfs-x509-helper tickets.

File: rpm/cvmfs-config-osg.spec [hunk 1]
```diff
@@ -22,6 +22,8 @@ Provides: oasis-config = 9
 
 Conflicts: cvmfs-config-default
 
+Requires: cvmfs >= 2.6.0
```
Comment (DrDaveD) [inline rpm/cvmfs-config-osg.spec:25]: No.  As I said in the ticket, we won't be having a Requires in cvmfs-config-osg.  cvmfs and cvmfs-config-osg will continue to have "=" in osg-oasis.  Only cvmfs-x509-helper will be ">=" in osg-oasis.

**Review by DrDaveD — CHANGES_REQUESTED** [review]
Please close this PR.

File: rpm/cvmfs-config-osg.spec (hunk 1 above)
Comment (efajardo) [reply rpm/cvmfs-config-osg.spec:25]: @DrDaveD what is the problem on having a version requirement on cvmfs-config-osg?

Comment (DrDaveD) [reply rpm/cvmfs-config-osg.spec:25]: It's just one more thing to maintain and doesn't get us anything beyond the setting in osg-oasis.  Even if somebody didn't have osg-oasis installed, I don't expect anybody to do an upgrade of cvmfs-config-osg without upgrading cvmfs.  In fact the newer cvmfs-config-osg does work with older cvmfs versions.  There is also a pending update to cvmfs-config-osg [SOFTWARE-3736](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3736) that I am holding off until cvmfs-2.6.1 which is expected any day.

**Pull request closed without merging**